Month3/LinkedList/LinkedList/TDLList.h

```diff
@@ -0,0 +1,404 @@
+#include <iostream>
+
+template<typename T>
+
+class TDLList
+{
+private:
+	struct FNode
+	{
+		T Element;
+		FNode* Next = nullptr;
+		FNode* Prev = nullptr;
```

Comment:
I'd encourage you not to shorten the key word of a variable name, Previous is less prone to confusions

Reply:
you shouldn't shorten the names